components/tasks/createScreen.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component name="createScreen" extends="Task">
```

Comment:
```suggestion
<component name="CreateScreen" extends="Task">
```

Reply:
What is the purpose of this task?

Reply:
to generate a new screen to put on top of the main screen

Reply:
We should create the main screen only one in the main thread. Please remove this task and use normal fields to change the main screen background.